README.md

```diff
@@ -54,7 +52,7 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+- [❓ FAQ](#faq)
```

Comment:
- Hey, since we've removed this section from the readme body, let's remove it also from the readme navigation to ensure users do not click links that are broken and do not exits in the readme file.

Reply:
done